api/index.mdx

```diff
@@ -26,6 +26,7 @@ change pganalyze settings.
 
 - [Creating an API key](/docs/api/create-api-key)
 - [Queries](/docs/api/queries)
+    * [getServers - Get a list of servers for an organization](/docs/api/queries/getServers)
```

Comment:
sorry this is unrelated but just noticed that it's not listed in https://pganalyze.com/docs/api so I wanted to add it